See https://github.com/fish-shell/fish-shell/issues/11036

disable the tests observed to be flaky for now.

based on https://github.com/fish-shell/fish-shell/commit/1835f141eb4ee271e1e845fcf42338f12cb3913d

diff --git a/tests/pexpects/bind.py b/tests/pexpects/bind.py
index fdbe9b629..4c535c97a 100644
--- a/tests/pexpects/bind.py
+++ b/tests/pexpects/bind.py
@@ -4,6 +4,10 @@ import os
 import platform
 import sys
 
+import os, sys
+if "CI" in os.environ:
+    sys.exit(127)
+
 # Skip on macOS on Github Actions because it's too resource-starved
 # and fails this a lot.
 #
diff --git a/tests/pexpects/commandline.py b/tests/pexpects/commandline.py
index 571cec309..e7d1862c1 100644
--- a/tests/pexpects/commandline.py
+++ b/tests/pexpects/commandline.py
@@ -1,6 +1,10 @@
 #!/usr/bin/env python3
 from pexpect_helper import SpawnedProc, control
 
+import os, sys
+if "CI" in os.environ:
+    sys.exit(127)
+
 sp = SpawnedProc()
 send, sendline, sleep, expect_prompt, expect_re, expect_str = (
     sp.send,
diff --git a/tests/pexpects/fkr.py b/tests/pexpects/fkr.py
index 4864d9200..fe12e1f72 100644
--- a/tests/pexpects/fkr.py
+++ b/tests/pexpects/fkr.py
@@ -3,6 +3,10 @@ from pexpect_helper import SpawnedProc
 from time import sleep
 import os
 
+import os, sys
+if "CI" in os.environ:
+    sys.exit(127)
+
 os.environ["fish_escape_delay_ms"] = "10"
 sp = SpawnedProc()
 send, sendline, sleep, expect_prompt, expect_re, expect_str = (
